assignment.js

```diff
@@ -112,8 +123,45 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
-
+let userChoice = prompt("Which one do you choose, rock, paper, or scissors?")
+let computerChoice = Math.random()
+if (computerChoice < 0.34) {
+    computerChoice = "rock"
+} else if (computerChoice <= 0.67) {
+    computerChoice = "paper"
+} else {
+    computerChoice = "scissors"
+}
+
+let compare = function (player1Choice, player2Choice) {
+    if (player1Choice === player2Choice) {
+        return "the result is a tie"
+    }
+    if (player1Choice === "rock") {
+        if (player2Choice === "scissors") {
+            return "rock wins"
+        } else {
+            return "paper wins"
+        }
+    }
+    if (player1Choice === "paper") {
+        if (player2Choice === "rock") {
+            return "paper wins"
+        } else {
+            return "scissors wins"
+        }
+    }
+    if (player1Choice === "scissors") {
+        if (player2Choice === "rock") {
+            return "rock wins"
+        } else {
+            return "scissors wins"
+        }
+    }
+};
+
+console.log(compare(userChoice, computerChoice));
 // Add your code above this line
 
```

Comment:
@OcielC Great Job here on creating this compare function to check for the winning outcomes. Also, nice way to create a way to generate a random choice for the computer. I would suggest adding one more conditional that would return an error if the `user` inputs the wrong choice or spells the choices wrong.
Take a look at this method to the problem [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L129)